app/menu.js

```diff
@@ -32,7 +32,11 @@ export default class MenuBuilder
         {
             this.setupDevelopmentEnvironment();
         }
-
+        
+        if ( isRunningDebug )
```

Comment:
@shriram2301 here you need to check we're not using `spectron`.

Also, I wonder if here is the best place to be triggering devTools opening and not in `openWindow` `setupBgWindow` files for example?

Reply:
in `openWindow` I believe the `menuBuilder` initializes an object of type `MenuBuilder` which is used to create the browser menus by taking in various arguments, this is the class exported by `menu`. I figured since all the logic for stuff like `isHot` is already being used there makes sense to add the stuff for debug in the same place. I will check for `spectron` just to be sure and run it as well. For the `bg` I will restrict all changes to only `setupBgWindow`